exposure-notification/src/main/resources/db/migration/V02.01__create_statistics_tables.sql

```diff
@@ -0,0 +1,4 @@
+create table en.stats_diagnosis_keys_created (
```

Comment:
This is a slightly misleading name, as it logs post-events and each event may create 0-14 keys.

Reply:
Yep, it refers to api, not actual keys, but I'll make better name.

Reply:
Changed